### PR TITLE
Adding description for the rdf file

### PR DESCRIPTION
**Pull request by delapuente**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,140 @@
...
+  "basePath": "awesome",
+  "versions": ["v1"]
+}
+```
+
+Then the resource collection `"cars"` will be available through `awesome/v1/cars`.
+
+### `common`
+
+Configures common aspects of the answes provided by the server. In particular:
```
Comment (frsela) [inline README.md]: s/answes/answers

File: README.md [hunk 2]
```diff
@@ -1,12 +1,140 @@
...
+}
+```
+
+Each version is a map with endpoints as keys and endpoint descriptors as values. For instance:
+
+```javascript
+{
+  "v1": {
+    "cars": { /*...*/ }
+    "cars/:carId": { /*...*/ }
```
Comment (frsela) [inline README.md:91]: we should explain that :xxx is a named variable

File: README.md [hunk 3]
```diff
@@ -1,12 +1,140 @@
...
+##### `params`
+
+Describe the expected parameters for this request to be valid. It is a map of parameter names as keys and types as values. For instance:
+
+```javascript
+{
+  "licensePlate": "string",
+  "picture": "file"
+}
+```
```
Comment (frsela) [inline README.md:142]: Add an example

**Comment by frsela** [comment]
awesome ! thank you.
r+ with nits ;)

File: README.md (hunk 3 above)
Comment (delapuente) [inline README.md:142]: It is an example already.

**Pull request merged**